### PR TITLE
Update about to reflect blog, not broader us-rse

### PR DESCRIPTION
**Pull request by cosden**
I've taken a first pass at updating the about page to reduce the scope to just the blog. 

This is to address issue #9

### CODE REVIEW

File: pages/about.md [hunk 1]
```diff
@@ -1,34 +1,13 @@
...
-description: About the United States (US) Research Software Engineer Community
+title: About US-RSE Syndicated Blog
+description: About the United States (US) Research Software Engineer Community Syndicated Blog
 permalink: /about/
 ---
 
-The increasing use of digital technologies across research communities has gone hand
-in hand with a strong growth and reliance on software written or customized to solve
-research problems. This in turn is driving dialog around opportunities to
+The US-RSE community blog collects articles and blog posts from personal and group RSE sites associated with the [US-RSE](https://us-rse.org) organization.  By collecting and aggregating the posts we hope to provide a single resource highlighting the work, stories and experiences of RSEs throughout the community.  Topics range from posts outline new RSE-developed software releases, to opinion pieces on the state of research software, to community updates and news, and everything in between.  The content is all generated and published by the individuals and groups owning the blogs.
```
Comment (vsoch) [inline pages/about.md]: > Topics range from posts *outlining*

File: pages/about.md [hunk 2]
```diff
@@ -1,34 +1,13 @@
...
-other titles are increasingly found as core roles in research teams.
-
-Initiatives in the [UK](http://rse.ac.uk/), [Germany](http://www.de-rse.org/en), the
-[Netherlands](http://nl-rse.org) and elsewhere are bringing
-together the community of people writing and contributing to research software at
-the national and international level. In the US this encompasses
-universities, laboratories, knowledge institutes, companies and other
-enterprises. This site is part of that effort.
+Disclaimer:
+Blog feeds are curated nightly to collect new posts and archive old posts.  As a result, we cannot guarantee that anyone has reviewed any posts for content or applicability.  Opinions expressed within each post are those of the author(s) and do not necessarily reflect the opinions of the US-RSE.  Offensive or inappropriate posts should be reported to blog@us-rse.org or a pull request should be submitted to remove the offending feed.
```
Comment (vsoch) [inline pages/about.md]: > A pull request should be *submit*

**Comment by cosden** [comment]
Is the disclaimer ok?  I'm just trying to be proactively careful.

**Comment by vsoch** [comment]
Yes I think it's a great idea - it's spot on that we can't control incoming content (other than the selection of the author) and that's exactly what should be done if there is something flagged.

**Pull request merged**